tock/tock/templates/hours/timecard_form.html

```diff
@@ -202,6 +253,12 @@ <h3 class="usa-alert-heading">{{ timecard_note.title }}</h3>
     };
 
     $("#save-timecard").on("click", function() {
+      // Clear anything saved locally.  The server is the
+      // source of truth.
+      if(window.localStorage) {
+        window.localStorage.removeItem('tock-entered-hours');
+      }
+
```

Comment:
One last piece change needed!  We probably want to clear the `localStorage` if a user clicks submit too, since that is also a save action. :-)